### ISSUE TITLE
How to disable "update to version 7" reminder?

### ISSUE THREAD

**Issue by paulwwisl** [csstools/postcss-plugins#1065]
### What would you want to propose?

In our project I see "Please update to version 7" everytime I change/save every file,
as development goes on with project running `npm start`

```

┌─────────────────────────────────────────────────────────────────────────────────┐
│                                                                                 │
│   This version of postcss-preset-env is not optimised to work with PostCSS 8.   │
│                Please update to version 7 of PostCSS Preset Env.                │
│                                                                                 │
│                    If you find issues, you can report it at:                    │
│          https://github.com/csstools/postcss-plugins/issues/new/choose          │
│                                                                                 │
└─────────────────────────────────────────────────────────────────────────────────┘
```
That clutters console output, taking more than 50% of it.

I would like to complete disable such,  
or show them only once per day,

### Suggested solution

 Configuration option remoundAboutUpdate true/false  would do.

### Additional context

npm has

`npm config set update-notifier false`

that is recommended as we manage node and npm via NVM.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/csstools/postcss-plugins/blob/master/CODE_OF_CONDUCT.md)
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by romainmenke** [comment]
Hi @paulwwisl,

Where you able to resolve this?

**Comment by paulwwisl** [comment]
Thanks for pointing to exact version.
It is larger team project, I am not to decide, and don't want to bring thing topic to meeting.

I would prefer to solve it for my computer first.
And then maybe come to meeting with solution (not with problem).

**Comment by romainmenke** [comment]
> And then maybe come to meeting with solution

For the migration from `6.7.0` to `7.x` we have a tool that might help you : 
https://www.npmjs.com/package/postcss-preset-env-migrate-6.7.0-to-7

This should make the migration extremely smooth.

_It is possible that you don't even need this tool if you don't use any of the three plugins bundled there._

-----

For 7.x to 8.x the situation is more complex.
We fixed a lot of long standing issues that were holding everything back.

We don't have tool for the migration because that wasn't possible.

We do however have an extensive Wiki : https://github.com/csstools/postcss-plugins/wiki/PostCSS-Preset-Env-8

------

8.x to 9.x is trivial.

We only changed the license : https://preset-env.cssdb.org/blog/license-change/

**Comment by romainmenke** [comment]
Hi @paulwwisl,

I am going to close this issue because there isn't anything we can or should do to resolve this. Please let use know if you do have any issues with migrating in this or a new issue.

**Comment by paulwwisl** [comment]
The question was how to disable reminder.

Nothing was possible or was added.

**Comment by romainmenke** [comment]
> The question was how to disable reminder.

You can pin to `6.7.0` or migrate to a later version.

Only `6.7.1` has the reminder.

**Comment by Antonio-Laguna** [comment]
And to be extra clear, that's the only thing that `6.7.1` contains. We added that reminder to encourage people to merge as this would normally be installed with `^6.0.0` so this new version would be installed and we hoped it would alert people about this (it did)

So you get the same CSS output with `6.7.0` as you do with `6.7.1`
